Source/NSWindow.m

```diff
@@ -723,6 +724,24 @@ + (void) initialize
     }
 }
 
++ (instancetype) windowWithContentViewController: (NSViewController *)viewController
+{
+  NSView *view = [viewController view];
+  NSRect frame = [view frame];
+  NSString *title = [viewController title];
+  NSUInteger style = NSTitledWindowMask |
+    NSClosableWindowMask |
+    NSMiniaturizableWindowMask |
+    NSResizableWindowMask;
+  NSWindow *window =  [[self alloc] initWithContentRect: frame
+                                              styleMask: style
+                                                backing: NSBackingStoreBuffered
+                                                  defer: NO];
+  [window setTitle: title];
+  [window setContentView: view];
+  return window;
```

Comment:
There seems to be an autorelease missing here.

Reply:
done.